tools/conventional-changelog-spectrum/templates/header.hbs

```diff
@@ -0,0 +1,29 @@
+{{#if isPatch~}}
+  ##
+{{~else~}}
+  #
```

Comment:
Not sure if I like the fact that patch releases get a smaller heading... Thoughts from others?

Reply:
I like small heading as it is more clear at the first glance.